compatibility-suite/tests/Service/MessagePactWriterInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace PhpPactTest\CompatibilitySuite\Service;
+
+use PhpPact\Config\PactConfigInterface;
+
+interface MessagePactWriterInterface
+{
+    public function write(string $name, string $body, string $consumer = 'c', string $provider = 'p', string $mode = PactConfigInterface::MODE_OVERWRITE): void;
```

Comment:
guessing the constants here are outside of control to make enums

(consumer = c and provider = p)

Reply:
This code look bad. I will try to improve by creating a DTO in a future PR.

Reply:
Create a PR to fix this hard coding https://github.com/pact-foundation/pact-php/pull/433